morituri/common/simplefreedb.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+# -*- Mode: Python; test-case-name: morituri.test.test_common_program -*-
+# vi:si:et:sw=4:sts=4:ts=4
+
+# Morituri - for those about to RIP
+
+# Copyright (C) 2017 Clément Bœsch <u@pkh.me>
+
+# This file is part of morituri.
```

Comment:
whipper!

Reply:
Yep, I'm in the process of rebasing that stuff, it was morituri when I wrote that patchset. I won't forget to fix that :)